make/validate.sh

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env bash
```

Comment:
Reminder to provide on error the suggested output and existing variable (if set), basically introduce "struct" as @boeboe suggested before to have a proper variables management.

Reply:
Done :heavy_check_mark: